jck/jtrunner/JavaTestRunner.java

```diff
@@ -893,9 +893,10 @@ public static boolean jckConfigurationForMultijvmWithNoAgent() throws Exception
 			extraJvmOptions += getTestSpecificJvmOptions(jckVersion, tests);
 
 			extraJvmOptions += suppressOutOfMemoryDumpOptions;
+			
 
 			// Add the JVM options supplied by the user plus those added in this method to the jtb file option.
-			fileContent += "set jck.env.devtools.refExecute.otherOpts \" " + extraJvmOptions + " \"" + ";\n";	
+			fileContent += "set jck.env.devtools.refExecute.otherOpts \" " + extraJvmOptions + " " + jvmOpts + " \"" + ";\n";	
```

Comment:
https://github.com/adoptium/aqa-tests/blob/6c6b9a5faaee7d1f8a2a3fb861fde3a1c554533b/jck/jtrunner/JavaTestRunner.java#L733 should be updated as well.